app/assets/stylesheets/publisher.scss

```diff
@@ -243,14 +237,6 @@
     .hide-location { display: inline-block; }
     .locator { display: none; }
   }
-
-  .twitter-typeahead {
-    left: -1px;
-    // Override inline rule of Typeahead
-    // scss-lint:disable ImportantRule
-    position: absolute !important;
```

Comment:
Well, adding the above code to the publisher css fixes the problem. It doesn't fix it for the comment textarea, but as I wrote above I also don't have a nice solution here, so the current behavior for the comment textarea looks ok to me.